packages/kokoas-server/src/handleRequest/reqSendContractDirectV2/generateContractPdfV2.ts

```diff
@@ -463,32 +464,49 @@ export const generateContractPdfV2 = async (
   const companyY2 = 197;
   const companyLH = payLineHeight; // 行の高さ。 今支払いとあわせていますが、変わる可能性
 
-  [companyY, companyY2].forEach((newY) => {
-
-    // 会社名　上下
-    drawText(
-      firstPage,
-      companyName,
-      {
-        x: companyX,
-        y: newY,
-        font: msChinoFont,
-      },
-    );
+  // 会社名　上
+  drawText(
+    firstPage,
+    companyName,
+    {
+      x: companyX,
+      y: companyY,
+      font: msChinoFont,
+    },
+  );
 
-    // 会社住所 上下
-    drawText(
-      firstPage,
-      companyAddress,
-      {
-        x: companyX,
-        y: newY - companyLH,
-        font: msChinoFont,
-      },
-    );
+  // 会社住所 上下
+  drawText(
+    firstPage,
+    companyAddress,
+    {
+      x: companyX,
+      y: companyY - companyLH,
+      font: msChinoFont,
+    },
+  );
 
+  // 会社名　下
+  drawText(
+    firstPage,
+    companyAddress,
+    {
+      x: companyX,
+      y: companyY2,
+      font: msChinoFont,
+    },
+  );
```

Comment:
```suggestion
  // 会社住所　下
  drawText(
    firstPage,
    companyAddress2,
    {
      x: companyX,
      y: companyY2,
      font: msChinoFont,
    },
  );
```

契約書下部の住所表記は、「住所」+「ハウスドゥ 〇〇〇〇店」とする、とのことでしたので、
```companyAddress```の中身自体を変更する必要があるかと思います。